diff --git a/web/ideacamp/src/app/feature/project-site/components/project-posts/project-posts.ts b/web/ideacamp/src/app/feature/project-site/components/project-posts/project-posts.ts
index c9bd925e..07ffe73f 100644
--- a/web/ideacamp/src/app/feature/project-site/components/project-posts/project-posts.ts
+++ b/web/ideacamp/src/app/feature/project-site/components/project-posts/project-posts.ts
@@ -33,7 +33,7 @@ export class ProjectPosts implements OnChanges, OnDestroy {
   posts = signal<ProjectPostResponse[]>([]);
   isLoading = signal(false);
   isCreating = signal(false);
-  errorMessage = signal<string | null>(null)
+  errorMessage = signal<string | null>(null);
 
   visibleCount = signal(3);
 
